Buildscripts/Flashing/merge.ps1

```
@@ -0,0 +1,20 @@
+param(
+    $port
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'port' has been declared but not used.

(PSReviewUnusedParameter)